filament/src/components/LightManager.cpp

```diff
@@ -253,13 +253,11 @@ void FLightManager::setIntensity(Instance i, float intensity) noexcept {
                 break;
 
             case Type::FOCUSED_SPOT: {
-                // li = lp / (2*pi*(1-cos(outer/2)))
+                // li = lp / (2 * pi * (1 - cos(outer / 2)))
```

Comment:
If we pass the half angle this comment should be updated, or at least augmented explaining that "outer" is the full angle of the spot

Reply:
oh yeah I see. I took this comment as the "real formula"